config/routes.rb

```diff
@@ -8,7 +8,12 @@
   devise_for :users, skip: :all
   root to: 'batch_contexts#new'
   resources :batch_contexts, only: %i[create index new show]
-  resources :job_runs, only: [:create, :index, :show]
-  get 'job_runs/:id/download_log', to: 'job_runs#download_log', as: 'download_log'
-  get 'job_runs/:id/download_report', to: 'job_runs#download_report', as: 'download_report'
+  resources :job_runs, only: [:create, :index, :show] do
+    member do
+      get :download_log
+      get :download_report
+      get :discovery_report_summary
+      get :progress_log
+    end
+  end
```

Comment:
added new routes and moved a couple existing ones (which changed their names slightly)